src/JobHelper.groovy

```diff
@@ -81,8 +81,8 @@ class JobHelper {
                     context.println "[SUCCESS] We have a response!\n${jsonResponse}"
                     break
                 } else {
-                    context.println "[RETRYWARNING] API Request was successful but jsonResponse is null. Retrying in 30 seconds..."
-                    context.sleep 30000
+                    context.println "[RETRYWARNING] API Request was successful but jsonResponse is null. Retrying in 60 seconds..."
```

Comment:
This says 60, but code is 30

Reply:
There is another retry underneath that sleeps the application for another 30 seconds after this